web/src/reducers/navigation.js

```diff
@@ -1,7 +1,7 @@
 import { NAVIGATION_SCROLL_TO_WAYPOINT } from '../actions/navigation';
 
 const initialState = {
-  activeSection: null
+  activeSection: 'apd-state-profile-overview'
```

Comment:
Default the active section of the nav bar to the state profile overview.  The `SideBar` component requires an active section be set, so...  set one.

Reply:
This raises a larger question about whether it _should_ require an active section to be set, IMO. But I guess we're only going to be using the sidebar on the builder page for now, so... maybe it's okay? (Having it hardcoded like this makes me a little uncomfortable, even though I've done the same thing elsewhere in the app.)

Reply:
We could also set a `defaultProp` on the sidebar itself.  Then at least this hard-coded value would be in the same place as its hard-coded counterparts?